views/wyr-single.ejs

```diff
@@ -79,17 +82,47 @@
         </li>
         <% }) %>
       </ul>
-      <form action="submit">
+      <form id="addCommentForm" action="submit">
         <textarea id="comment" class="border border-gray-200 rounded-lg p-2 w-full mt-4"
           placeholder="Add a comment"></textarea>
         <button id="add-comment"
           class="bg-green-500 text-white rounded px-4 py-2 mt-4 hover:bg-green-600 active:bg-green-700">Add
           Comment</button>
       </form>
-      <button id="close-modal"
+      <button id="addComment-close-modal"
+        class="bg-red-500 text-white rounded px-4 py-2 mt-4 hover:bg-red-600 active:bg-red-700">Close</button>
+    </div>
+  </div>
+
```

Comment:
changed the above names for id's according to the wyr-single.js files for the comments